homeassistant/components/cover/intent.py

```diff
@@ -23,3 +30,17 @@ async def async_setup_intents(hass: HomeAssistant) -> None:
             INTENT_CLOSE_COVER, DOMAIN, SERVICE_CLOSE_COVER, "Closed {}"
         ),
     )
+    intent.async_register(
+        hass,
+        intent.ServiceIntentHandler(
+            intent.INTENT_SET_POSITION,
+            DOMAIN,
+            SERVICE_SET_COVER_POSITION,
+            extra_slot_names={ATTR_POSITION},
+            extra_slot_schema={
+                vol.Required(ATTR_POSITION): vol.All(
+                    vol.Coerce(int), vol.Range(min=0, max=100)
```

Comment:
why coerce to int ?

Reply:
I copied the schema from the cover's set position service call. I suppose it doesn't matter since it'll be coerced there anyways.